src/engraving/playback/renderers/arpeggiorenderer.cpp

```diff
@@ -47,27 +47,28 @@ void ArpeggioRenderer::doRender(const Ms::EngravingItem* item, const mpe::Articu
         return;
     }
 
-    int stepsCount = chord->notes().size();
-    mpe::percentage_t percentageStep = mpe::HUNDRED_PERCENTS / stepsCount;
+    size_t stepsCount = chord->notes().size();
```

Comment:
I suggest to just change this to `int stepsCount = static_cast<int>(chord->notes().size());` and not change anything else

Reply:
OK